### ISSUE TITLE
Is the project still active?

### ISSUE THREAD

**Issue by netopolit** [Git-Sum/obsidian-enhance-youtube-links#4]
@Git-Sum, I'd like to submit a PR if the project is still actively maintained. Otherwise, I'd like to fork the repository, keep all the current functionality, add a couple of options and improvements, and maintain it.

I'd also like to review the PR from @1C0D and possibly approve it.

Thanks,
Mislav

**Comment by Git-Sum** [comment]
Hello, 

I will not be maintaining this project. Feel free to take it over.

**Comment by netopolit** [comment]
Hi,
Would you like me to use the repo as a base for a new plugin with a different name, or would you rather transfer the ownership? I don't have a preference either way.

**Comment by 1C0D** [comment]
at least one answer...

**Comment by timothyausten** [comment]
@netopolit If you want to fork to a new project, I'll put it to use.

**Comment by netopolit** [comment]
@timothyausten The main reason I wanted to work on this plugin is title fetching. In the meantime, I found this plugin https://github.com/zolrath/obsidian-auto-link-title that suits my needs. I don't think there will be added value in continuing work on [obsidian-enhance-youtube-links](https://github.com/Git-Sum/obsidian-enhance-youtube-links)

**Comment by timothyausten** [comment]
Looks like it captures YouTube video names, too.

**Comment by timothyausten** [comment]
Nope, it doesn't. Not on mobile anyway.

**Comment by netopolit** [comment]
It does on desktop, but I haven't tried it on mobile

**Comment by netopolit** [comment]
Found an open issue for mobile. Not sure if author will decide to fix it https://github.com/zolrath/obsidian-auto-link-title/issues/46

**Comment by dawn0us** [comment]
@netopolit You can check out plugins like "Thumbnails" and "Rich Links" which both work on mobile.

**Comment by netopolit** [comment]
@dawn0us Thank you. I was looking for a plugin that populates the video title on YouTube link paste (just a part of what this plugin did). I ended up using an exclamation mark to show the video in the MD file and [obsidian-auto-link-title](https://github.com/zolrath/obsidian-auto-link-title) to fetch the video title.

**Comment by dawn0us** [comment]
@netopolit I tried that already. But it doesn't work on link like this https://youtu.be/KHVR587oW8I?si=CbgpL9aZXm4sjsWZ (copied from share button). It only fetches "Youtube" as the title. How did you make it work?

**Comment by netopolit** [comment]
@dawn0us Strange, it works for me. At least, it does on the desktop (MacOS). I get the full title after pasting your link in youtu.be format.
![](https://github.com/user-attachments/assets/2defac87-65c0-4180-b6dc-1cff25523106). 

Are you on the desktop version of Obsidian? Check if you still have [obsidian-enhance-youtube-links](https://github.com/Git-Sum/obsidian-enhance-youtube-links) plugin active. It should be disabled or removed and [obsidian-auto-link-title](https://github.com/zolrath/obsidian-auto-link-title) installed and enabled.

**Comment by dawn0us** [comment]
@netopolit Thanks. It probably no longer works on mobile.

**Comment by netopolit** [comment]
@dawn0us You are right. I tried it now. I also found this issue: https://github.com/zolrath/obsidian-auto-link-title/issues/46 . The plugin was last updated four months ago, so there's still a chance this project is active, and the issue may be addressed in future version 🤞 .

**Comment by Git-Sum** [comment]
Closing this as others found alternatives and I will try and fix things as I can.
